modules/repository-s3/src/yamlRestTest/resources/rest-api-spec/test/repository_s3/20_repository_permanent_credentials.yml

```diff
@@ -10,12 +10,11 @@ setup:
         body:
           type: s3
           settings:
-            bucket: @permanent_bucket@
+            bucket: bucket
             client: integration_test_permanent
-            base_path: "@permanent_base_path@"
+            base_path: base_path_integration_tests
```

Comment:
Just making sure I understand this: These yaml tests do not run against 3rd party services so that passing these variables from the build.gradle file was never really useful for them. So we can just hard code them?

Reply:
Correct.